lib/components/common/timeline/list_timeline/widgets/list_timeline.dart

```diff
@@ -81,6 +81,16 @@ class _TopRow extends StatelessWidget {
                 name: name,
               ),
             ),
+            horizontalSpacer,
+            HarpyButton.raised(
+              padding: config.edgeInsets,
+              elevation: 0,
+              backgroundColor: theme.cardTheme.color,
+              icon: cubit.filter != TimelineFilter.empty
+                  ? Icon(Icons.filter_alt, color: theme.colorScheme.primary)
+                  : const Icon(Icons.filter_alt_outlined),
+              onTap: Scaffold.of(context).openEndDrawer,
```

Comment:
I just remembered why I haven't added the list timeline filter yet - this doesn't work when you have a list timeline in your home screen since the end drawer in the home screen contains the home timeline drawer.

There is no easy fix for it since the home screen scaffold lives above the home tab view so we can't change out the end drawer while we are on a different tab.

We can still merge it to master but I will wait for #341 until releasing it.

Reply:
Ah, I see what you mean. If only there were a way to dynamically swap which content is displayed in the endDrawer based on its context. i.e. `context.widget is HomeTimeline ? ListTimelineFilterDrawer() : HomeTimelineFilterDrawer()` 😕